### PR TITLE
Fixed broken absolute link

### PR DESCRIPTION
**Pull request by n-boshnakov**
**What this PR does / why we need it**:
This PR fixes a broken absolute link.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Release note**:

```doc developer
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -383,8 +383,6 @@ ingress:
     ...
 ```
 
-You can add [annotations](https://github.com/gardener/garden-setup/blob/master/components/ingress-controller/chart/templates/00-ingress-controller.yaml#L249) for the ingress controller load balancer service. This can be used for example to deploy an internal load balancer on your cloud provider (see the example for aws above).
```
Comment (Kristian-ZH) [inline README.md:386]: Hmm in my opinion now this is the correct link: https://github.com/gardener/garden-setup/blob/master/components/ingress-controller/deployment.yaml#L72

Comment (Kristian-ZH) [reply README.md:386]: /needs second-opinion

Comment (Diaphteiros) [reply README.md:386]: Oh, good catch. This link is from the times when we had the nginx chart as a part of garden-setup, now it is checked out during the deployment.

I think it is probably the best idea to keep the sentence (as it is still true) and just remove the link - most people anyway know how annotations work.

What do you think?

Comment (Kristian-ZH) [reply README.md:386]: lgtm

Comment (Kristian-ZH) [reply README.md:386]: Sounds good to me

Comment (n-boshnakov) [reply README.md:386]: That sounds like a good idea to me. I can make the changes now, if we're all in agreement?

**Review by Kristian-ZH — APPROVED** [review]
/lgtm

**Review by Diaphteiros — APPROVED** [review]

**Pull request merged**